helm/prometheus-agent/charts/prometheus-agent/templates/remotewritesecrets.yaml

```diff
@@ -0,0 +1,13 @@
+{{- range $remoteWrite := .Values.remoteWrite }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    {{- include "labels.common" $ | nindent 4 }}
+  name: {{ $remoteWrite.name }}-remote-write-api
+  namespace: {{ $.Release.Namespace }}
+stringData:
+  username: {{ $remoteWrite.username }}
```

Comment:
So, you will bind the secret in the values?

Reply:
PMO will create the config secret, so i think the value is base64?

Reply:
The secret will be in the values yes but we configure it from a secret anyway. I'm changing to data instead of string data as the value should be encoded in base64

Reply:
app-operator will read this from config secret, so they are base64 encoded! 
Have you tried this?

Reply:
I did test it and it is working on ginger. The app operator will see the decoded value when reading so I'm using stringData to ensure it is encoded in base64